cypress/integration/volunteer/generate_court_report.spec.js

```diff
@@ -8,7 +8,7 @@ context("Logging into cypress as a volunteer", () => {
   it("should generate a court report", () => {
     cy.get("#toggle-sidebar-js").click();
     cy.contains("Generate Court Reports").click();
-    cy.get("#case-selection").select("CINA-18-1003 - non-transition");
+    cy.get("#case-selection").select("CINA-18-1003 - transition");
```

Comment:
This change is unrelated. The seed data just happened to reach age 16 yesterday 😄